package.json

```diff
@@ -7,22 +7,26 @@
     "install": "ts-patch install",
     "compile": "ttsc",
     "start": "ttsc && node ./dist/sample.js",
-    "test": "mocha 'test.specs.ts' --require ts-node/register"
+    "test": "mocha './test/*.specs.ts' --require ts-node/register"
   },
   "author": "Merlin Goulet <merlin.goulet@live.fr>",
   "license": "MIT",
   "devDependencies": {
-    "typescript": "^3.0.0",
+    "@types/chai": "^4.2.7",
+    "@types/chai-as-promised": "^7.1.2",
+    "@types/mocha": "^5.2.7",
+    "ts-patch": "^1.2.4",
     "ts-toolbelt": "^6.9.4",
     "ttypescript": "^1.5.9",
-    "ts-patch": "^1.2.4",
-    "ts-node": "^8.0.0"
+    "mocha": "^6.2.2",
+    "fast-check" : "^3.0.0"
   },
   "optionalDependencies": {
-    "typescript": "^3.0.0",
     "ts-toolbelt": "^6.9.4",
-    "ttypescript": "^1.5.9",
-    "ts-patch": "^1.2.4",
-    "ts-node": "^8.0.0"
+    "ttypescript": "^1.5.9"
+  },
+  "dependencies": {
```

Comment:
There was no dependencies at the start and there should be no one at the end. Think as a user, you don't want to install a lib if it means you can have incoherence with your existing libs.

Reply:
Ok